Documentation.md

```diff
@@ -2,31 +2,62 @@
-| `cblite cp`    | Replicate, import or export a database |
-| `cblite help`  | Display help text |
-| `cblite info`  | Display information about the database |
-|`cblite logcat` | Converts binary log files to plain text |
-| `cblite ls`    | List the documents in the database |
-| `cblite put`   | Create or update a document |
-| `cblite query` | Run queries, using the [JSON Query Schema][QUERY] |
-|`cblite reindex` | Rebuilds indexes, which may improve performance |
-| `cblite revs`  | List the revisions of a document |
-| `cblite rm`    | Delete a document |
-| `cblite select`| Run queries, using [N1QL][N1QL] syntax |
-| `cblite serve` | Start a (rudimentary) REST API listener |
+In one-shot usage the first argument is a subcommand name, followed by optional flag arguments and then (usually) a database path. For example:
+
+```sh
+$ cblite info travel-sample.cblite2
+```
+
+Or you can run the tool in an interactive mode, like a cute little special-purpose shell, by omitting the subcommand name:
+
+```sh
+$ cblite travel-sample.cblite2
+```
+
+# Sub-Command List
+
+Later sections document each subcommand and its specific parameters. 
+
+| Subcommand     | Purpose                                   |
+|----------------|-------------------------------------------|
+| `cat`, `get`   | Display the body of one or more documents |
+| `cd`           | Set the current collection |
+| `check`        | Check the database file for corruption |
+| `compact`      | Compact the database file, freeing up disk space ✍️ |
+| `cp`           | Replicate, import or export a database |
+| `decrypt`      | Remove encryption from a database 👔 ✍️ |
+| `encrypt`      | Encrypt or rekey a database 👔 ✍️ |
+| `edit`         | Update or create a document as JSON in a text editor ✍️ |
+| `export`       | Copy documents to a JSON file |
+| `help`         | Display help text |
+| `import`       | Import documents from a JSON file ✍️ |
+| `info`, `file` | Display information about the database |
+| `ls`           | List the documents in the current collection |
+| `lscoll`       | List the collections in the database |
+| `mkcoll`       | Create a collection ✍️ |
+| `mkindex`      | Create an index ✍️ |
+| `mv`           | Move documents to a different collection ✍️ |
+| `openremote` | Pull a remote DB to a temp file and open it in interactive mode |
+| `push` | Replicate changes to a remote database |
+|  `pull` | Replicate changes from a remote database ✍️ |
+| `put`          | Create or update a document ✍️ |
+| `query`        | Run queries, using the [JSON Query Schema][QUERY] |
+| `reindex`      | Rebuild indexes, which may improve performance ✍️ |
+| `revs`         | List the revisions of a document |
+| `rm`           | Delete documents ✍️ |
+| `rmindex`      | Remove an index ✍️ |
+| `select`       | Run queries, using [SQL++][N1QL] syntax |
+| `serve`        | Start a (rudimentary) REST API listener |
```

Comment:
serve should be removed right?

Reply:
Yup. Made some other minor updates to that file too.